casaos-fix-docker-api-version/README.md

```
@@ -171,16 +173,23 @@ If you want to allow Docker upgrades in the future (after confirming CasaOS comp
 sudo apt-mark unhold docker-ce docker-ce-cli containerd.io
 ```
 
-⚠️ **Warning for LXC users**: Before unholdingcontainerd.io, check if the AppArmor issue has been fixed:
+⚠️ **Warning for LXC/Proxmox users**: Before unholding containerd.io, check if the AppArmor issue has been fixed:
 - Check: https://github.com/opencontainers/runc/issues/4968
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)